quiz/backend/players/tests/test_views.py

```diff
@@ -0,0 +1,17 @@
+from players.views import NicknameGeneratorAPIView
+
+
+def test_nickname_generator_view_returns_correct_data(api_rf):
+    request = api_rf.get("/players/nickname-generator/")
```

Comment:
API diagram uses different endpoint

Reply:
@ToJestKrzysio do You think we could rename this endpoint to `nicknames/` or sth like that? I'm just asking because I have this feeling that `username/` is a bit misleading in the context of what we are doing here.

Reply:
guess we could, username sounded pretty clear to me.